docker-vertica-v2/Dockerfile

```diff
@@ -44,10 +53,17 @@ RUN set -x \
   openssl \
   && /usr/sbin/groupadd -r verticadba \
   && /usr/sbin/useradd -r -m -s /bin/bash -g verticadba dbadmin \
-  && yum localinstall -y /tmp/${VERTICA_RPM} \
+  && ls /tmp/ \
+  && if [[ "$TARGETARCH" == "arm64" ]] ; then \
+      uname -m && \
+      yum localinstall -y /tmp/${VERTICA_ARM64_RPM} ; \
+    elif [[ "$TARGETARCH" == "amd64" ]] ; then \
+      yum localinstall -y /tmp/${VERTICA_X86_RPM} ; \
+    else \
+      exit 1; \
+    fi \
```

Comment:
At the end there will be one image compatible with multiple architectures, right? How is the build process going to handle the fact that we have 2 rpms?

Reply:
Correct. We will need to download both x86 and aarch64 rpms to the packages directory. We can do the same in Jenkins as well for daily, private and public image builds.
The build is agnostic really, if we set TARGETARCH to `linux/arm64,linux/amd64`, it would be expected that both the RPMs are available in the packages/. Otherwise the docker build will fail. If we provide any one of them, it should still pass and build a single arch image, as long as the matching RPM is present.
Although, I expect that moving forward our default behavior will be multi-arch only.